diff -Naur -x '*~' -x '*.orig' -x '*.rej' syslog-ng-3.4.3/contrib/systemd/syslog-ng.service syslog-ng-3.4.3-service-configuration/contrib/systemd/syslog-ng.service
--- syslog-ng-3.4.3/contrib/systemd/syslog-ng.service	2013-08-13 11:25:18.000000000 +0200
+++ syslog-ng-3.4.3-service-configuration/contrib/systemd/syslog-ng.service	2013-08-19 10:33:40.504315326 +0200
@@ -4,7 +4,8 @@
 
 [Service]
 Sockets=syslog.socket
-ExecStart=/usr/sbin/syslog-ng -F
+EnvironmentFile=-/etc/sysconfig/syslog-ng
+ExecStart=/sbin/syslog-ng $OPTIONS
 ExecReload=/bin/kill -HUP $MAINPID
 StandardOutput=null
 Restart=on-failure
